oci/config/convert/runtime.go

```diff
@@ -123,31 +126,23 @@ func MutateRuntimeSpec(g rgen.Generator, rootfs string, image ispec.Image, manif
 
 	g.SetProcessUID(uint32(execUser.Uid))
 	g.SetProcessGID(uint32(execUser.Gid))
-	g.ClearProcessAdditionalGids()
```

Comment:
We almost certainly need this because otherwise if the default from `runtime-spec` changes for some reason you might end up with supplementary group IDs you don't expect.